Harry Branch

HarryOTO opened the pull request:
Added Commands
Updated Simulation
Added turret soft limits
Updated ToF library

HarryOTO committed:
Updated TimeOfFlight Library
2/3/24 16:11

KeeganRubingh committed:
Set up commands and bound to buttons.
2/3/24 -  16:19

HarryOTO committed:
Fixed Turret Sim Soft Limits
2/3/24 17:26

HarryOTO committed:
Merge branch 'Harry-Branch' of https://github.com/FRC930/Robot2024 into Harry-Branch

KeeganRubingh committed:
Command comments, Mechanism2d setpoints.
2/3/24 - 17:45

HarryOTO committed:
Merge remote-tracking branch 'origin/master' into Harry-Branch

jackrschumacher reviewed (changes requested):
Thinking about what we are doing with initialization methods using SmartDashboard, since we are only using that for testing.

wegnergit committed:
Create Test Versions of command just for testing
8:00PM
Test classes can obsolete after done testing and remove configTestBinding()
and smartdashboard values are centralized in commands

HarryOTO committed:
Merge remote-tracking branch 'origin/master' into Harry-Branch

HarryOTO committed:
Moved Test Commands to New Folder and Removed Unused Imports
2/5/24 17:55

HarryOTO committed:
Implemented SetElevatorPositionCommandTest
2/5/24 18:04

KeeganRubingh committed:
Robot control test refactor
19:22 - 2/5/24
Split test controls off driver to codriver controller
Added driving controls back so driving sim possible
fixed elevator sim

KeeganRubingh committed:
Made test positional commands not reset pose and removed redundant isFinished override.
19:27  - 2/5/24

HarryOTO committed:
Implemented Status Code retries
2/5/24 20:46

HarryOTO committed:
Merge branch 'Harry-Branch' of https://github.com/FRC930/Robot2024 into Harry-Branch

jackrschumacher committed:
(2/6/24 17:48) Removed unused imports in robotContainer and changed the name of configureDriverBindings

jackrschumacher commented on src/main/java/frc/robot/commands/SparkTestShooterCommand.java:
Lets get rid of Spark

jackrschumacher committed:
(18:03 2/6/23) Update Comments and adjust code in ElevatorType for readability

jackrschumacher committed:
(18:11 2/6/24) Update comments to give attribution in Phoenix6Utility

jackrschumacher reviewed (approved):
Reviewed by Jack, Matt and Steve
